sdk/include/vcruntime/msc/intrin.h

```diff
@@ -380,6 +380,10 @@ void _sgdt(void *Destination);
 #pragma intrinsic(__sev)
 #endif
 
+#if defined _M_ARM64
```

Comment:
```suggestion
#if defined(_M_ARM64)
```
(keeping the ambient style)